tests/installation/bootloader_s390.pm

```diff
@@ -495,33 +153,18 @@ sub run() {
     eval {
         ###################################################################
         # connect to zVM, login to the guest
-        my $reconnect = exists get_var("DEBUG")->{"no get_to_yast"};
-        $r = $s3270->connect_and_login($reconnect);
-
-        $self->get_to_yast() unless $reconnect;
+        $r = $s3270->connect_and_login();
+        $self->get_to_yast();
```

Comment:
I would suggest to keep the former approach valid deciding if 'get_to_yast' should be called or if a reconnect is tried. @Soulofdestiny and me try to add the functionality to skip tests, e.g. like qemu snapshot with SKIPTO. Either you merge this first and we add if/else afterwards or you keep this open for now.